protocol.c

```diff
@@ -68,7 +67,7 @@ static uint8_t protocol_execute_line(char *line)
   }
 
   /* If there was an error, report it */
-  if (status && status != STATUS_IDLE_WAIT) {
+  if (status != STATUS_IDLE_WAIT) {
```

Comment:
'NULL' status would be OK, in which case we still wanted to go ahead and report it (which is how it was before, but I just want to avoid IDLE status)